packages/web/src/scss/components/FileUploader/_theme.scss

```diff
@@ -1,27 +1,32 @@
-@use '@tokens' as tokens;
+@use '@global' as global-tokens;
+@use '../../settings/globals';
 
-$row-gap: tokens.$space-600;
+$row-gap: global-tokens.$space-700;
 
-$input-drop-zone-padding: tokens.$space-600;
-$input-drop-zone-column-gap: tokens.$space-600;
-$input-drop-zone-border: tokens.$border-width-100 tokens.$border-style-100 tokens.$border-primary-default;
-$input-drop-zone-draggable-border: tokens.$border-width-100 tokens.$border-style-200 tokens.$border-primary-default;
-$input-drop-zone-dragging-border: tokens.$border-width-100 tokens.$border-style-200 tokens.$border-primary-selected;
-$input-drop-zone-border-color-hover: tokens.$border-primary-hover;
-$input-drop-zone-border-color-active: tokens.$border-primary-active;
-$input-drop-zone-border-radius: tokens.$radius-100;
-$input-drop-zone-background: tokens.$background-basic;
-$input-drop-zone-label-typography: tokens.$body-medium-text-regular;
-$input-drop-zone-label-color: tokens.$text-primary-default;
-$input-drop-zone-icon-color: tokens.$action-link-primary-default;
+$input-drop-zone-padding: global-tokens.$space-700;
+$input-drop-zone-column-gap: global-tokens.$space-700;
+$input-drop-zone-border: global-tokens.$border-width-100 solid var(--#{globals.$prefix}color-form-field-border-default);
+$input-drop-zone-draggable-border: global-tokens.$border-width-100 dashed
```

Comment:
Dropzone border should be 2px thick. Definitely in draggable state, maybe in the click-only state too.
![image](https://github.com/user-attachments/assets/74657301-cfcc-40d6-b492-bdec8881d587)

Reply:
switched to border-width-200 everywhere